recipes/freexl/all/test_package/test_package.c

```diff
@@ -1,317 +1,10 @@
-    return -43;
-  }
-
-  ret = freexl_get_cell_value(handle, 3, 3, &cell_value);
-  if (ret != FREEXL_OK) {
-    fprintf(stderr, "Error getting cell value (3,3): %d\n", ret);
-    return -44;
-  }
-  if (cell_value.type != FREEXL_CELL_DATE) {
-    fprintf(stderr, "Unexpected cell (3,3) type: %u\n", cell_value.type);
-    return -45;
-  }
-  if (strcmp(cell_value.value.text_value, "1967-10-01") != 0) {
-    fprintf(stderr, "Unexpected cell (3,3) value: %s\n", cell_value.value.text_value);
-    return -46;
-  }
-
-  ret = freexl_get_cell_value(handle, 3, 4, &cell_value);
-  if (ret != FREEXL_OK) {
-    fprintf(stderr, "Error getting cell value (3,4): %d\n", ret);
-    return -47;
-  }
-  if (cell_value.type != FREEXL_CELL_INT) {
-    fprintf(stderr, "Unexpected cell (3,4) type: %u\n", cell_value.type);
-    return -48;
-  }
-  if (cell_value.value.int_value != 4) {
-    fprintf(stderr, "Unexpected cell (3,4) value: %d\n", cell_value.value.int_value);
-    return -49;
-  }
-
-  ret = freexl_get_cell_value(handle, 3, 5, &cell_value);
-  if (ret != FREEXL_OK) {
-    fprintf(stderr, "Error getting cell value (3,5): %d\n", ret);
-    return -50;
-  }
-  if (cell_value.type != FREEXL_CELL_INT) {
-    fprintf(stderr, "Unexpected cell (3,5) type: %u\n", cell_value.type);
-    return -51;
-  }
-  if (cell_value.value.int_value != 237) {
-    fprintf(stderr, "Unexpected cell (3,5) value: %d\n", cell_value.value.int_value);
-    return -52;
-  }
-
-  ret = freexl_get_cell_value(handle, 1, 5, &cell_value);
-  if (ret != FREEXL_OK) {
-    fprintf(stderr, "Error getting cell value (1,5): %d\n", ret);
-    return -53;
-  }
-  if (cell_value.type != FREEXL_CELL_TIME) {
-    fprintf(stderr, "Unexpected cell (1,5) type: %u\n", cell_value.type);
-    return -54;
-  }
-  if (strcmp(cell_value.value.text_value, "23:34:04") != 0) {
-    fprintf(stderr, "Unexpected cell (1,5) value: %s\n", cell_value.value.text_value);
-    return -55;
-  }
+#include "freexl.h"
 
-  /* error cases */
-  ret = freexl_get_cell_value(handle, 7, 3, &cell_value);
-  if (ret != FREEXL_ILLEGAL_CELL_ROW_COL) {
-    fprintf(stderr, "Unexpected result for (7,3): %d\n", ret);
-    return -56;
-  }
-  ret = freexl_get_cell_value(handle, 2, 99, &cell_value);
-  if (ret != FREEXL_ILLEGAL_CELL_ROW_COL) {
-    fprintf(stderr, "Unexpected result for (2,99): %d\n", ret);
-    return -57;
-  }
-  ret = freexl_get_cell_value(handle, 4, 2, &cell_value);
-  if (ret != FREEXL_ILLEGAL_CELL_ROW_COL) {
-    fprintf(stderr, "Unexpected result for (4,2): %d\n", ret);
-    return -58;
-  }
-  ret = freexl_get_cell_value(handle, 3, 6, &cell_value);
-  if (ret != FREEXL_ILLEGAL_CELL_ROW_COL) {
-    fprintf(stderr, "Unexpected result for (3,6): %d\n", ret);
-    return -59;
-  }
 
-  ret = freexl_close(handle);
-  if (ret != FREEXL_OK) {
-    fprintf(stderr, "CLOSE ERROR: %d\n", ret);
-    return -2;
-  }
+int main() {
+    const void *handle;
+    int ret;
+    ret = freexl_open(NULL, &handle);
```

Comment:
This sends `NULL` to `fopen` which is UB afaik, should probably find another function :)

Reply:
Passing a nonexistent file path will probably work just fine here. It will simply return an error code, which can be ignored.